### PR TITLE
Javascript2 week1

### PR DESCRIPTION
**Pull request by melo4yxa77796**
my memo game

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            <div id="time"></div>
+        </div>
+        <div class="game-container"></div>
+        
+        <button id="stop" class="hide">Stop game</button>
+        
+    </div>
+    <div class="controls-container">
+        <p id="results"></p>
+        <button id="start" onclick="window.location.reload();">Start game</button>
```
Comment (vdaguenet) [inline index.html]: Prefer adding events handler in the JS file instead of inline code.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const moves = document.getElementById("moves-count");
+const timeValue = document.getElementById("time");
+const startButton = document.getElementById("start");
+const stopButton = document.getElementById("stop");
+const gameContainer = document.querySelector(".game-container");
+const result = document.getElementById("result");
+const controls = document.querySelector("controls-container");
+let cards;
+let interval;
+;
```
Comment (vdaguenet) [inline index.js]: There is a wild semicolumn on the loose 😱

File: index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+const emojis=["🥶","🥶","😈","😈","👿","👿","😡","😡","🥵","🥵","🤬","🤬","🎃","🎃","🤮","🤮","😰","😰","😨","😨"];
+let shuf_emojis = emojis.sort(function() {
+    return (Math.random() > 0.5) ? 1 : -1;
+});
+
+ for( let i = 0; i<emojis.length; i++){
```
Comment (vdaguenet) [inline index.js]: Be careful with the formatting of your code
```suggestion
 for(let i = 0; i < emojis.length; i++){
```

File: index.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    let box = document.createElement('div');
+    box.className = 'item';
+    box.innerHTML = shuf_emojis[i];
+    box.onclick = function() {
+        this.classList.add('boxOpen')
```
Comment (vdaguenet) [inline index.js]: The requirements for this week include `on click, the card flips from showing the backside pattern to the picture and vice versa.` Your project lacks the functionality to flip the card back.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    background: #f4c531;
+    transition: 0.25s;
+    transform: rotateY(0deg);
+    backface-visibility: hidden;
+}
+.boxOpen::after{
+    transform: rotateY(180deg);
+}
+
+@media (max-width:600px) {
```
Comment (vdaguenet) [inline style.css]: Please make sure to indent your code properly.

**Review by vdaguenet — CHANGES_REQUESTED** [review]
It's a good start. However, there is a feature missing.

Also, be mindful of the indentation and use of spaces and empty lines in your code.

File: index.html (hunk 1 above)
Comment (melo4yxa77796) [reply index.html]: hi
I changed

File: index.js (hunk 2 above)
Comment (melo4yxa77796) [reply index.js]: I am sorry i forgot 😊

File: index.js (hunk 3 above)
Comment (melo4yxa77796) [reply index.js]: i formatted it

File: style.css (hunk 5 above)
Comment (melo4yxa77796) [reply style.css]: i don't really understand here, but i formatted it with prettier

File: index.js (hunk 4 above)
Comment (melo4yxa77796) [reply index.js]: i don't know how to do it but i hope figure it out

**Comment by melo4yxa77796** [comment]
Thanks for comments

File: index.js (hunk 4 above)
Comment (vdaguenet) [reply index.js]: In your `onClick` function, you need to check whether the box is open. If yes, you close it. If not, you open it. Check out the functions available for `classList` here: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList; you might find something useful.

Comment (melo4yxa77796) [reply index.js]: hi again i did it can you check please?

File: index.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+});
+
+let shuf_emojis = emojis.sort(function () {
+  return Math.random() > 0.5 ? 1 : -1;
+});
+
+for (let i = 0; i < emojis.length; i++) {
+  let box = document.createElement("div");
+    box.className = "item";
+  box.innerHTML = shuf_emojis[i];
```
Comment (vdaguenet) [inline index.js:49]: Be careful with indentation. You sometimes use 2 spaces, and at other times, you use 4 spaces. Choose one and stick with it. It is very important to be mindful of indentation to make your code easy to understand for both your coworkers and yourself.

**Review by vdaguenet — APPROVED** [review]

File: index.js (hunk 4 above)
Comment (vdaguenet) [reply index.js]: Very good 👍🏻

File: index.js (hunk 6 above)
Comment (melo4yxa77796) [reply index.js:49]: okay thank you

**Pull request merged**